WooCommerce/Classes/ViewRelated/Dashboard/Settings/About/AboutViewController.swift

```diff
@@ -22,7 +22,7 @@ class AboutViewController: UIViewController {
             Bundle.main.detailedVersionNumber()
         )
         let localizedTitleTextLine2 = String.localizedStringWithFormat(
-            NSLocalizedString("© %@ Automattic, Inc.", comment: "About View's Footer Text. The variable is the current year"),
+            NSLocalizedString("© %@ WooCommerce, Inc and WooCommerce Ireland Ltd.", comment: "About View's Footer Text. The variable is the current year"),
```

Comment:
I think there should be a period after `"Inc"`. 

```suggestion
            NSLocalizedString("© %@ WooCommerce, Inc. and WooCommerce Ireland Ltd.", comment: "About View's Footer Text. The variable is the current year"),
```

Reply:
Thanks! I originally copied this text directly from the team requesting the change, and it didn't include a period, so I left it as it was written. But I just double checked the Chicago Manual of Style (which our house style is based on) and confirmed that they always include the period in the abbreviations `Inc.` and `Ltd.` so I agree we should add it here.